### PR TITLE
ActorSystem naming should work on Java 11

### PR DESCRIPTION
**Pull request by bpholt**

### CODE REVIEW

File: specs2-akka/src/main/scala/com/dwolla/testutils/akka/AkkaTestKitSpecs2Support.scala [hunk 1]
```diff
@@ -13,5 +13,9 @@ class AkkaTestKitSpecs2Support(config: Option[Config] = None)(implicit execution
   def after: Unit = TestKit.shutdownActorSystem(system)
 
   private def safeTypeName: String = enclosingClassName(this.getClass)
-  @tailrec private def enclosingClassName(clazz: Class[_]): String = if (clazz.isLocalClass) enclosingClassName(clazz.getEnclosingClass) else clazz.getSimpleName
+  @tailrec private def enclosingClassName(clazz: Class[_]): String =
+    if (clazz.isLocalClass || clazz.isAnonymousClass)
```
Comment (bpholt) [inline specs2-akka/src/main/scala/com/dwolla/testutils/akka/AkkaTestKitSpecs2Support.scala:17]: On Java 8, `clazz.isLocalClass` returns true, but on Java 11, it returns false. The opposite seems to be true for `clazz.isAnonymousClass`. Using `||` to combine them should work on either platform.

**Review by calebwsutton — APPROVED** [review]

File: project/Dependencies.scala [hunk 2]
```diff
@@ -6,7 +6,7 @@ object Dependencies {
 
   val SjsMacroTaskExecutor = Def.setting("org.scala-js" %%% "scala-js-macrotask-executor" % "1.0.0")
   val SjsSecureRandom = Def.setting("org.scala-js" %%% "scalajs-java-securerandom" % "1.0.0")
-  val logback = "ch.qos.logback" % "logback-classic" % "1.3.5"
+  val logback = "ch.qos.logback" % "logback-classic" % "1.2.11"
```
Comment (aquilesg) [inline project/Dependencies.scala:9]: Why did we downgrade on the `logback-classic` version?

File: specs2-akka/src/main/scala/com/dwolla/testutils/akka/AkkaTestKitSpecs2Support.scala (hunk 1 above)
Comment (aquilesg) [reply specs2-akka/src/main/scala/com/dwolla/testutils/akka/AkkaTestKitSpecs2Support.scala:17]: Is there a specific reason why this is the case? It seems like such a weird thing to change in between versions.

File: project/Dependencies.scala (hunk 2 above)
Comment (bpholt) [reply project/Dependencies.scala:9]: Because I didn't want to force users into accepting an slf4j version change

**Pull request merged**